ds-prep-python-tuples/main.py

```diff
@@ -0,0 +1,26 @@
+passenger = (12, True, "Bonnell, Miss. Elizabeth", "female", 58)
+print(passenger)
+name = passenger[2]
+age = passenger[-1]
+print(name, age)
+survived_and_name = passenger[1:3]
+gender_and_age = passenger[3:]
+print(survived_and_name, gender_and_age)
+is_female = "female" in passenger
+is_male = "male" in passenger
+print(is_female, is_male)
+def get_survival_info(passenger):
+	(ID, survived, name, gender, age) = passenger
+	passenger = (age, gender, survived)
+	print(passenger)
```

Comment:
It's better to have your functions `return` their value, and have the outside code print it, as it makes your functions re-useable:

---

![image](https://user-images.githubusercontent.com/16143771/164943628-836ab689-e335-4c19-aca8-e733a29471f9.png)